src/common/sim.ts

```diff
@@ -3,7 +3,14 @@ import Level from './map'; // alias to not conflict with a map collection
 import State from './state';
 import { Body, Box, Circle, Vec2, World } from 'planck-js';
 import { Enemy, Entity, Player } from './entity';
-import { MOVEMENT_SPEED } from './constants';
+import {
+  MOVEMENT_SPEED,
+  PLAYER_HEALTH_MAX,
+  PLAYER_HITBOX_X,
+  PLAYER_HITBOX_Y,
+  PLAYER_SPAWN_X,
+  PLAYER_SPAWN_Y,
+} from './constants';
```

Comment:
```suggestion
import * as constants from './constants';
```
så slipper vi importera allt ett för ett